data/sql/updates/pending_db_world/rev_1631616272860535120.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631616272860535120');
+
+UPDATE creature SET wander_distance = 5 WHERE guid = 56058;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56059;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56086;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56127;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56160;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56162;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56125;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56123;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56166;
+UPDATE creature SET wander_distance = 5 WHERE guid = 56150;
```

Comment:
Add backticks and make this one query please https://www.azerothcore.org/wiki/sql-standards#compact-queries

Reply:
Also when working in the creature table it is preferred if you update guid AND id